site/content/en/docs/Advanced/multi-cluster-allocation.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Multi-cluster Allocation"
+date: 2019-10-25T05:45:05Z
+description: >
+  In order to allow allocation from multiple clusters, Agones provides a mechanism to set redirect rules for allocation requests to the right cluster.
+---
+
+There may be different types of clusters, such as on-premise, and Google Kubernetes Engine (GKE), used for a game to help with the cost saving and availability. To plan for the cluster usage and boost availability, Agones provides a mechanism to define priorities on the clusters. Priorities are defined on [GameServerAllocationPolicy](https://github.com/googleforgames/agones/blob/master/pkg/apis/multicluster/v1alpha1/gameserverallocationpolicy.go) agones CRD. A match maker can enable the multi-cluster rules on a request and target [agones-allocator]({{< relref "allocator-service.md">}}) endpoint in any of the clusters and get resources allocated on the cluster with the highest priority. If the cluster with highest priority is overloaded, the allocation request is redirected to the cluster with the next highest priority.
+
+To enable multi-cluster allocation on game servers, here are the steps to be taken:
+
+1. Set cluster priorities
+2. Establish trust between agones-allocator services in all clusters
+3. Allocate with multi-cluster feature enabled
+
+## Define Cluster Priority
+
+[GameServerAllocationPolicy](https://github.com/googleforgames/agones/blob/master/pkg/apis/multicluster/v1alpha1/gameserverallocationpolicy.go) is the CRD defined by Agones for setting multi-cluster allocation rules. In addition to cluster priority, it describes the connection information for the target cluster, including the game server namespace, agones-allocator endpoint and client K8s secrets name for redirecting the allocation request. Here is an example of setting the priority for a cluster and it's connection rules. One of such resource should be defined per cluster. For clusters with the same prioiry, the cluster is chosen with a probability relative to its weight.
+
+In the following example the policy is defined for cluster B in cluster A.
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: multicluster.agones.dev/v1alpha1
+kind: GameServerAllocationPolicy
+metadata:
+  name: allocator-cluster-B
+  namespace: cluster-A-ns
+spec:
+  connectionInfo:
+    allocationEndpoints:
+    - 34.82.195.204
+    clusterName: "clusterB"
+    namespace: cluster-B-ns
+    secretName: allocator-client-to-cluster-B
+  priority: 1
+  weight: 100
+EOF
+```
+
+For a local cluster allocation rule, allocationEndpoints should remain unset.
```

Comment:
This part isn't clear. Does this mean I also need to create a policy in cluster A to allocate to itself (cluster A) with a weight and priority?